satellite/audit/verifier.go

```diff
@@ -535,7 +559,7 @@ OUTER:
 	return err
 }
 
-// checkIfSegmentDeleted checks if stripe's pointer has been deleted since stripe was selected.
+// checkIfSegmentDeleted checks if path's pointer has been deleted since path was selected.
```

Comment:
Note let's rename this to `checkSegmentAltered` rather than deleted, because the important part is whether it was changed in any way, not just deletion.